Add some explanatory text to the error message that occurs
if you do not have a suitable IWAD.  If we relax the 
doom-wad dependency to a Depends:, this may be necessary.

                            -- Jon Dowland <jon@alcopop.org>
--- a/src/d_main.c~	2006-11-19 17:44:42.000000000 +0000
+++ b/src/d_main.c	2008-07-21 17:49:22.000000000 +0100
@@ -841,7 +841,11 @@
     free(iwad);
   }
   else
-    I_Error("IdentifyVersion: IWAD not found\n");
+    I_Error("IdentifyVersion: IWAD not found\n"
+        "prboom requires a doom data (\"IWAD\") file to use.\n"
+        "please install one, either freedoom or a package built\n"
+        "using game-data-packager.\n"
+    );
 }
 
 
